docs/sharing-data.md

```diff
@@ -258,15 +258,31 @@ accessed through the variable namespace **terramate**.
 This can be referenced from any Terramate code to reference
 information like the path of the stack or its name.
 
-## terramate.root.path.fs.absolute (string)
+
+## Project Metadata
+
+Project metadata is the same independent of stack.
```

Comment:
This reads funny, did you move this section? maybe it would make more sense if defined after the stack metadata.

Reply:
This section didn't exist and I added it before the stack because on my mind it would be project wide -> stack specific, but I could change the order, no strong opinion :+1: